Gem/Code/Source/Components/CharacterComponent.cpp

```diff
@@ -119,7 +119,7 @@ namespace MultiplayerSample
             }
         }
 
-        if (!HasController())
+        if (!HasController() || GetController()->IsAutonomous())
```

Comment:
If we’re autonomous we don’t need to be calling setBasePosition on the character. We do the setBasePosition call as part of process input, since that’s the only place we should ever be writing predictive state for an autonomous.

Reply:
This is actually specifically for applying a correction which is the only time we do need to. It comes through the same code path we'd see for an entity update which happens to be this.

Reply:
This *might* go away with some of changes Sergey is working on. The issue here is the order of ops between competing systems trying to update transform causes the correction to get blown out. This corrects the other systems to what we need so everything stays consistent.